<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PicturesPainterSetAspectRatioClass</class>
 <widget class="QDialog" name="PicturesPainterSetAspectRatioClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PicturesPainterSetAspectRatio</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: rgb(161, 161, 161);
    padding: 2px; 
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">QLabel {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #f8fafc, stop:1 #e3e7ed); /* 浅灰渐变 */
    border: 2px solid #b5c0cd;             /* 淡蓝边框 */
    border-radius: 12px;                   /* 圆角 */
    padding: 6px;                          /* 内边距，防图片紧贴边缘 */
    color: #555;                           /* 文字颜色（如有文字时） */
    font-weight: bold;
    font-size: 16px;
}</string>
            </property>
            <property name="text">
             <string>图像宽度:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_imgWidth">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 15px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">QLabel {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #f8fafc, stop:1 #e3e7ed); /* 浅灰渐变 */
    border: 2px solid #b5c0cd;             /* 淡蓝边框 */
    border-radius: 12px;                   /* 圆角 */
    padding: 6px;                          /* 内边距，防图片紧贴边缘 */
    color: #555;                           /* 文字颜色（如有文字时） */
    font-weight: bold;
    font-size: 16px;
}</string>
            </property>
            <property name="text">
             <string>图像高度:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_imgHeight">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 15px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_ok">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 30px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
            </property>
            <property name="text">
             <string>确定</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_exit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 30px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
